<dec f='boost-1.85.0/libs/contract/include/boost/contract/core/exception.hpp' l='347' type='boost::contract::from_failure_handler boost::contract::exception_::get_except_failure_locked()'/>
<use f='boost-1.85.0/libs/contract/include/boost/contract/core/exception.hpp' l='677' u='c' c='_ZN5boost8contract18get_except_failureEv'/>
<def f='boost-1.85.0/libs/contract/include/boost/contract/detail/inlined/core/exception.hpp' l='314' ll='317' type='boost::contract::from_failure_handler boost::contract::exception_::get_except_failure_locked()'/>
